### PR TITLE
Change Start a Call button to DefaultButton

### PR DESCRIPTION
**Pull request by ralphgabrinao**
# What

Change dropdown button to a regular button

# Why

There is only one option where a Rooms call is started as a Presenter.

# How Tested
2022-12-08 13_51_02-defaultConfig json - communication-services-virtual-visits-js - Visual Studio Co

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) documentation.

**Does this PR contain ARM Template changes?**

- [ ] I have verified deploying using the ARM Template
- [ ] I have tested the deployed app end to end

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: client/src/components/home/HomeComponent.tsx [hunk 1]
```diff
@@ -61,59 +38,74 @@ export const HomeComponent = (props: HomeComponentProps): JSX.Element => {
...
+  const callCreateRoom = async (): Promise<void> => {
+    try {
+      const redirectUrl = await createRoomAndRedirectUrl();
+      window.location.assign(redirectUrl);
+    } catch (error) {
+      setError(error);
+    }
+  };
+
+  const HomeButton = ({ iconName, text, onClick }): JSX.Element => {
```
Comment (tejaswip-msft) [inline client/src/components/home/HomeComponent.tsx:63]: should we move this out or is it okay to leave it inside of the HomeButtons component

Comment (ralphgabrinao) [reply client/src/components/home/HomeComponent.tsx:63]: I'll move it out

**Review by tejaswip-msft — DISMISSED** [review]

File: client/src/components/home/HomeComponent.tsx [hunk 2]
```diff
@@ -61,59 +38,83 @@ export const HomeComponent = (props: HomeComponentProps): JSX.Element => {
...
+const HomeButton = ({ theme, iconName, text, onClick }): JSX.Element => {
+  const buttonStyles = getButtonStyles(theme);
+  return (
+    <DefaultButton styles={buttonStyles} iconProps={{ iconName }} onClick={onClick}>
+      <Text styles={buttonTextStyles}>{text}</Text>
+    </DefaultButton>
+  );
+};
+
+const LearnMore = (): JSX.Element => {
```
Comment (dailam-msft) [inline client/src/components/home/HomeComponent.tsx:89]: Better to move this to  ""https://github.com/Azure-Samples/communication-services-virtual-visits-js/blob/main/client/src/components/LearnMoreItem.tsx" I think and rename it to LearnMore.tsx. So related components will be together in a file, and HomeComponent.tsx will be shorter.

File: client/src/components/home/HomeComponent.tsx [hunk 3]
```diff
@@ -61,59 +38,83 @@ export const HomeComponent = (props: HomeComponentProps): JSX.Element => {
...
+    <>
+      <Text styles={font16pxStyle}>Learn more about Azure Communication Services</Text>
+      <LearnMoreItem
+        headerText={'Azure Communication Services virtual appointments'}
+        headerLink={'https://learn.microsoft.com/azure/communication-services/tutorials/virtual-visits'}
+        description={
+          'This tutorial describes concepts for virtual appointment applications. After completing this tutorial and the associated Sample Builder, you will understand common use cases that a virtual appointment delivers...'
+        }
+      />
+      <LearnMoreItem
```
Comment (dailam-msft) [inline client/src/components/home/HomeComponent.tsx:100]: As part of general refactoring we could also bring this file - ""https://github.com/Azure-Samples/communication-services-virtual-visits-js/blob/main/client/src/components/LearnMoreItem.tsx" -  into the components/home folder, so related files reside within same folder

**Review by tejaswip-msft — APPROVED** [review]

**Review by dailam-msft — APPROVED** [review]

**Pull request merged**